src/commands/run.rs

```diff
@@ -144,13 +144,30 @@ pub async fn command(args: Args, _json: bool) -> Result<()> {
         // this is for `rails c` and similar REPLs
     })?;
 
-    let exit_status =
-        tokio::process::Command::new(args.args.first().context("No command provided")?)
-            .args(args.args[1..].iter())
-            .envs(variables)
-            .status()
-            .await
-            .context("Failed to spawn command")?;
+    let slash_c = "/C".to_owned();
```

Comment:
There _has_ to be a better way to do this...

Down below, you could probably replace `&slash_c` with `"/C".into()`, right?

Reply:
I'll make the change during lunch break, I'm not so familiar with rust stuff yet. Thanks for the comment!

Reply:
Can't borrow the "/C".into() value since it is dropped too quickly. Instead, I just mapped args.args into a Vec<&str>.